### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by RobertRen1122**

### CODE REVIEW

File: aceknow/settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 SECRET_KEY = SECRET
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```
Comment (Louis-He) [inline aceknow/settings.py:28]: Leave it as true in development

File: frontend/package-lock.json [hunk 2]
```diff
@@ -1816,6 +1821,97 @@
         }
       }
     },
+    "@material-ui/core": {
```
Comment (Louis-He) [inline frontend/package-lock.json]: Do we have to use material-ui?

**Review by Louis-He — CHANGES_REQUESTED** [review]

**Comment by CatherineZM** [comment]
Do we need this pull request? Isn't nav bar already in search bar page?

**Comment by RobertRen1122** [comment]
> Do we need this pull request? Isn't nav bar already in search bar page?

Search bar branch was checked out from nav bar branch, we can close this pr. Cathy or I could clean up the code on the search bar pr.

**Comment by CatherineZM** [comment]
> > Do we need this pull request? Isn't nav bar already in search bar page?
> 
> Search bar branch was checked out from nav bar branch, we can close this pr. Cathy or I could clean up the code on the search bar pr.

@RobertRen1122 sounds good. I will close this pr and after you clean up the comments in search bar pr it's good to go.

**Pull request closed without merging**